model/textparse/nhcbparse_test.go

```diff
@@ -493,7 +494,6 @@ something_bucket{a="b",le="+Inf"} 9 # {id="something-test"} 2e100 123.000
 				{Labels: labels.FromStrings("id", "something-test"), Value: 0.5},
 				{Labels: labels.FromStrings("id", "something-test"), Value: 8.0},
 			},
-			// TODO(krajorama): ct:   int64p(1520430001000),
```

Comment:
note to reviewers: input doesn't have created timestamp, copy paste error to have this comment.